src/game/bosses/boss_8.c

```diff
@@ -0,0 +1,1525 @@
+#include "global.h"
+#include "core.h"
+#include "flags.h"
+#include "task.h"
+#include "trig.h"
+#include "bg_triangles.h"
+#include "malloc_vram.h"
+#include "gba/defines.h"
+#include "gba/io_reg.h"
+#include "lib/m4a.h"
+#include "sakit/globals.h"
+#include "sakit/camera.h"
+#include "sakit/collision.h"
+#include "sakit/player.h"
```

Comment:
I think we should be using Camera and Player from the stage headers, because that's where the global is actually defined for them? It also means you get all the related functions which seems logical?

Reply:
I don't quite know what you mean.
But in general I feel like include-files should not `#include` other files, unless it is necessary.
Separating them just makes everything a bit clearer.

But we definitely need to think about renaming "sakit" into something else.
The code is not a separate "entity" from the game.